docs/guides/medical.md

```diff
@@ -0,0 +1,181 @@
+- szansę, że otworzy się w tym czasie (jeśli się nie otworzy w tym czasie to nigdy się nie otworzy)
+- kolejno ile ran danej wielkości można załatać jednym bandażem
+
+| Efektywność | | + | ++ |
+|-----------------------|:----:|:----:|:-----:|
+| Czas otwierania (min) | 5-10 | 6-20 | 10-30 |
+| Szansa otwarcia | 0.4 | 0.3 | 0.1 |
+| Ef. Duże | 2 | 4 | 8 |
+| Ef. Średnie | 3 | 5 | 9 |
+| Ef. Małe | 4 | 6 | 10 |
+
+**Podsumowując:**
+- Rodzaj bandaża nie odgrywa takiej roli. Jeśli masz dostęp do szycia możesz bandażować się dowolnym bandażem.
+- Jeżeli sytuacja nie wymaga natychmiastowej pomocy medyka, nie wołaj go. Użyj staz, zabandażuj się, wtedy zapytaj medyka czy może Ci pomóc.
+
+
+## Krew
+
+Podczas rozgrywki krew jest jednym z najcenniejszych zapasów. Z tego powodu, w miarę możliwości trzymaj przy sobie krew i uzupełniaj jej zapas, jeżeli jest taka możliwość. 0,5 - 1,0 l w zupełności wystarczy, gdybyś stracił przytomność oraz znacząco odciążysz medyka. Jeżeli jesteś przytomny, nie potrzebujesz przetoczenia krwi do momentu pojawienia się szarego ekranu.
+
+## Środki przeciwbólowe
+
+### APAP
+
+W razie wystąpienia bólu, zażyj APAP. Jeżeli siła bólu jest duża,zażyj 2 tabletki APAP. Praktycznie nie powoduje on objawów ubocznych (chyba że zażyjesz 10 tabletek w bardzo krótkim czasie). Jego opóźnienie w działaniu jest dłuższe, a czas działania krótszy niż morfina, aczkolwiek siła tłumienia bólu jest niewiele mniejsza od morfiny.
+
+### Morfina
+
+Dużą zaletą morfiny jest szybkie i długotrwałe działanie, aczkolwiek podanie jej wiąże się z obniżeniem ciśnienia krwi oraz tętna, co w przypadku znacznej utraty krwi może doprowadzić do utraty przytomności. Najważniejszą rzeczą, na którą trzeba zwracać uwagę, to odstęp między kolejnymi dawkami. Przyjęcie w czasie mniejszym niż 30 minut (czasu wirtualnego) więcej niż 2 dawek najpewniej skończy się utratą przytomności.
+
+## Tętno i CPR
+
+Sprawdzając tętno upewnij się że nikt nie wykonuje akurat RKO i nie sprawdzasz tętna na kończynie ze stazą.
+
+Dla tętna przyjmuje się następujące wartości:
+- Niskie – poniżej 60
+- Normalne – 60 – 100
+- Wysokie – powyżej 100
+
+### Brak pulsu
+
+Rozpocznij wykonywanie RKO (CPR). Po każdym wykonaniu sprawdź czy nie wrócił puls. Jeśli pacjentem zajmuje się więcej osób to podczas sprawdzania pulsu upewnij się, że nikt nie wykonuje RKO (CPR), ponieważ możesz wyczuć puls, kiedy tak naprawdę dalej go nie ma. Jeśli nie jesteś medykiem wykonuj RKO (CPR), aż do przybycia pomocy lub odzyskania pulsu. Każdorazowa akcja reanimacyjna wydłuża czas do śmierci.
+
+## Słuchaj medyka
+
+Nie bez powodu medyk jest, po SL i TL, 3 osobą w kolejności dowodzenia. To on odpowiada przez całą misję za stan zdrowotny całej drużyny. Wykonuj każde jego polecenie. Wspomagaj go, kiedy o to poprosi, ale nie stój nad nim kiedy nie ma takiej potrzeby. 
+
+![Pierwsza pomoc](../_data/guides/medical/bpp_first_aid.png)
+
+# Medycy
```

Comment:
Only one first level heading can be present in file. Move all the subchapters one level lower too.
```suggestion
## Medycy
```